--- sivar/src/views/mejoramiento/siembra-campo/viveros/ViveroFormView.vue
+++ sivar/src/views/mejoramiento/siembra-campo/viveros/ViveroFormView.vue
@@ -296,7 +296,6 @@
                     :disabled="!form.proyecto_id"
                     :class="{ 'bg-slate-100 border-slate-200 text-slate-400 cursor-not-allowed shadow-inner': !form.proyecto_id }"
                   />
-                </div>
                 </div>
                   <div
                     v-if="showCaracteres && form.proyecto_id"
