--- sivar/src/views/mejoramiento/siembra-campo/viveros/ViveroFormView.vue
+++ sivar/src/views/mejoramiento/siembra-campo/viveros/ViveroFormView.vue
@@ -1818,7 +1818,7 @@
     form.value.caracteres_ids = [];
   }
   if (!form.value.caracteres_ids.includes(car.id)) {
-    form.value.caracteres_ids.push(car.id);
+    form.value.caracteres_ids = [...form.value.caracteres_ids, car.id];
   }
   searchCaracter.value = "";
 };
