--- api_sivar/app/Http/Controllers/ViveroController.php
+++ api_sivar/app/Http/Controllers/ViveroController.php
@@ -205,6 +205,10 @@
                 $vivero->save();
             }
 
+            if ($request->has('caracteres_ids')) {
+                $vivero->caracteres()->sync($request->caracteres_ids);
+            }
+
             return response()->json($vivero, 201);
         });
     }
@@ -367,6 +371,10 @@
 
             $viveroA->update($request->except(['lote_id', 'consecutivo_vivero_ingenio']));
 
+            if ($request->has('caracteres_ids')) {
+                $viveroA->caracteres()->sync($request->caracteres_ids);
+            }
+
             // Manage Lote History (either Lote or slot changed!)
             if ($oldLoteId != $newLoteId || $oldConsecutivo != $newConsecutivo) {
